DOCS/kratos-explanation-security.md

```diff
@@ -0,0 +1,28 @@
+# Kratos Charm Security
+
+This document provides cryptographic documentation for the Kratos charm. Its purpose is to track the exposure of charm code to cryptographic attack vectors.
+
+What is not included in this document and regarded as out of scope:
+
+- Workload code (refer to the workloads’ cryptographic documentation).
+- Data at rest encryption.
+
+## Sensitive Data Exchange
+
+The charm relies on Juju secrets:
+
+- To store Kratos admin password.
+- To store Kratos cookie secret that is used to encrypt session cookies.
+
+Github secrets are used during development, build, test and deploy phases:
+
+- To get Charmcraft credentials that are used to interact with Charmhub.
+- To get a Github token that is used to interact with Github API.
+
+## Cryptographic tech and packages in use
+
+Kratos charm uses the following cryptography packages:
+
+- Python secrets built-in library is used to generate Kratos cookie secret.
+
+Kratos charm relies on `receive-ca-cert` Juju interface and uses CA certificates.
```

Comment:
Let's clarify that this is the `ca-certificates` dpkg and that we rely on the mozilla cert bundle. Also I don't know if I would consider the `receive-ca-cert` charm lib a cryptographic tech, but feel free to keep it

Reply:
Removed mentioning of the relation.

Added context around `ca-certificates`. Please take a look if the links point to the right places.